src/meshapi/tests/test_nn.py

```diff
@@ -59,7 +63,7 @@ def test_nn_valid_install_number(self):
         self.assertEqual(
             expected_nn,
             resp_nn,
-            f"status code incorrect for test_nn_valid_install_number. Should be {code}, but got {response.status_code}",
+            f"nn incorrect for test_nn_valid_install_number. Should be {expected_nn}, but got {resp_nn}",
```

Comment:
Oh, uh, good catch?

Reply:
Haha yeah I copied this and it wasn't right